src/import/load-costume.js

```diff
@@ -31,7 +33,24 @@ const loadCostumeFromAsset = function (costume, costumeAsset, runtime) {
     if (costumeAsset.assetType === AssetType.ImageVector) {
         // createSVGSkin does the right thing if rotationCenter isn't provided, so it's okay if it's
         // undefined here
-        costume.skinId = renderer.createSVGSkin(costumeAsset.decodeText(), rotationCenter);
+        let svgString = costumeAsset.decodeText();
+        if (costume.version && costume.version === 2) {
+            // SVG Renderer load fixes "quirks" associated with Scratch 2 projects
+            const svgRenderer = new SvgRenderer();
+            svgRenderer.loadString(svgString);
+            svgString = svgRenderer.toString();
+            delete costume.version;
```

Comment:
As I understand it, costume.version is getting created for all sb2 costumes (vector and bitmap) but it's only getting deleted for the vector costumes. It should also get deleted for bitmap costumes (if it exists).

Reply:
Good catch! Let me know if this makes sense, or maybe we should pass it without messing around with the costume object.

Reply:
I think you might be right that it makes more sense to add an optional argument at the end of `loadCostume` that only ever gets passed in from sb2 costumes. I don't think any of the other arguments are optional, so I think we won't need to modify the other calls to loadCostume.
It's possible that this only works out because none of the costumes/backdrops from the library have text in them (is that true?)...

Is there any other way to identify, from the SVG data itself that it's a Scratch 2.0 SVG?

Reply:
I switched it to an argument.

I think SVGs from 2.0 can be identified by all of them containing a comment
<!-- Exported by Scratch - http://scratch.mit.edu/ -->

We might want to add a different one for 3.0